website/docs/docs/build/validation.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Validations
+id: validation
+description: "MetricFlow has three types of built-in validations, including Parsing Validation, Semantic Validation, and Data Warehouse validation, which are performed in a sequential and blocking manner."
+sidebar_label: "Validations"
+tags: [Metrics, Semantic Layer]
+---
+
+Validations refers to the process of checking whether a system or configuration meets the expected requirements or constraints. In the case of dbt and MetricFlow, there are three built-in validations &mdash; [parsing](#parsing), [semantic](#semantic), and [data platform](#data-platform). 
+
+These validations ensure that configuration files follow the expected schema, the semantic graph doesn't violate any constraints, and semantic definitions in the graph exist in the physical table - providing effective data governance support. These three validation steps occur sequentially and must succeed before proceeding to the next step.
+
+The validation code for [MetricFlow can be found here.](https://github.com/transform-data/metricflow/tree/main/metricflow/model/validations)
```

Comment:
this URL returns a page 404 - @Jstein77 do you have the correct URL?

Reply:
is it this one: https://github.com/dbt-labs/metricflow/tree/main/dbt_semantic_interfaces/tests/validations?

Reply:
Hey we moved the validation code into a new repo. This is the correct link https://github.com/dbt-labs/dbt-semantic-interfaces/tree/main/dbt_semantic_interfaces/validations